python/nano/src/bigdl/nano/pytorch/utils.py

```diff
@@ -58,15 +60,21 @@ class ChannelsLastCallback(pl.Callback):
     def setup(self, trainer, pl_module, stage: Optional[str] = None) -> None:
         """Override hook setup to convert model to channels_last and wrap DataHook."""
         # TODO: Add check for module_states
+        try:
```

Comment:
The weights_prepack optimization of ipex will change the rank of convolution's weights to 5, e.g., [64, 3, 3, 3] to [1, 3, 3, 3, 64],  which make `model.to(...)` fail.